### DISCUSSION TITLE
feature-request: Organisation special repository

### DISCUSSION THREAD

**Discussion by Eywek** [community/community#5055] [Other Feature Feedback, Questions, & Ideas]
For users account we're able to create "special repository" to add a README that will be visible on our public profile (i.e. github.com/eywek/eywek). That's a nice feature that could be extended to organizations.

Having a "special repository" for organizations could be useful to:
- Have a readme that is showed on the public page of the organization, where we could explain what's available on github, how the OSS process works, links to documentation or code of conduct...
- Have default configuration for any new repository (issues templates, license, code of conduct...)
- Have github actions workflows that could run when some or all repositories are updated, which can allow to enhance deployment process (i.e. when a commit is pushed on main branch in a repository of that organization, it will trigger a "organization" workflow which will deploy the commit to the preprod environnement)

**Comment by alimttth** [comment]
as for unlimited links, you can use the default README on your landing page to include all of your links, you can even make it beautiful by making a link icon instead of the actual link  this is what it looks like

**Comment by rickstaa** [comment]
@Eywek, I think https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile partly does what you are looking for. I see that default `CONTRIBUTING`, and `CODE_OF_CONDUCT` are also already supported using the `.github` repository (see https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#about-default-community-health-files).

**Comment by jckimble** [comment]
@Eywek you can add a profile directory in a repository named .github (don't forget the .) with a README.md to do the same as Readme on a personal page.
